src/Compilers/VisualBasic/Test/Semantic/Binding/BindingObjectInitializerTests.vb

```diff
@@ -1091,7 +1091,7 @@ End Class
         End Sub
 
         <Fact(), WorkItem(788522, "DevDiv")>
-        Sub ObjectInitializerNoStackOverflowFor250LevelsOfNesting()
+        Sub ObjectInitializerNoStackOverflowFor150LevelsOfNesting()
```

Comment:
I wouldn't say it's OK, but it's what happens...

It's unlikely we can do much here -- the parsing is inherently recursive and 64-bit frames are larger, so we overflow sooner.